### ISSUE TITLE
how to convert wall_time column

### ISSUE THREAD

**Issue by psteinb** [j3soon/tbparse#14]
When using the SummaryReader, I was experimenting with the `extra_column=set(["wall_time"])` feature. I just wanted to share that I was able to convert the timestamp that this produces like so:

``` python
from tbparse import SummaryReader
logdir = "/someplace/local"
reader_ = SummaryReader(logdir, extra_columns=set(['wall_time']))
df = reader_.scalars
df.head()
   step                 tag     value     wall_time
0     0  dice/train/average  0.162965  1.684921e+09
1     1  dice/train/average  0.046228  1.684921e+09
2     2  dice/train/average  0.008472  1.684921e+09
3     3  dice/train/average  0.006491  1.684921e+09
4     4  dice/train/average  0.122127  1.684921e+09
```
It took me some time to figure out how wall_time is encoded. But I managed:
``` python
df["wall_clock"] = pd.to_datetime(df.wall_time, unit="s")
df.head()
   step                 tag     value     wall_time                    wall_clock
0     0  dice/train/average  0.162965  1.684921e+09 2023-05-24 09:34:08.409187328
1     1  dice/train/average  0.046228  1.684921e+09 2023-05-24 09:34:13.541285888
2     2  dice/train/average  0.008472  1.684921e+09 2023-05-24 09:34:18.608008448
3     3  dice/train/average  0.006491  1.684921e+09 2023-05-24 09:34:23.644415232
4     4  dice/train/average  0.122127  1.684921e+09 2023-05-24 09:34:28.763384064
```
I think it is a great feature to extract the wall_time of a step/epoch to give users a rough estimate on how long each epoch took. I suggest that this could be documented a bit better. If you point me the way, I can contribute.

**Comment by j3soon** [comment]
Thanks for sharing this!

I agree that the `extra_columns` argument isn't well documented now. 

I just added a page [here](https://tbparse.readthedocs.io/en/latest/pages/extra-columns.html) for documenting the usages. If you are familiar with Sphinx & reStructuredText, you can open a pull request by editing this file: https://github.com/j3soon/tbparse/blob/master/docs/pages/extra-columns.rst

If you don't want to deal with the hassle of writing reStructuredText and building the docs locally with Sphinx, simply edit the file as if it were a Markdown file. I'll convert those to reStructuredText and build them locally before merging.

**Comment by psteinb** [comment]
Thanks a lot. See #15

**Comment by j3soon** [comment]
Thank you for the PR!

I just merged #15, closing this issue.
